src/main/webapp/index.html

```diff
@@ -1,10 +1,18 @@
 <!DOCTYPE html>
 <meta charset="UTF-8">
+<div id="navbar-container"></div>
 <title>Open Sesame</title>
 <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 <link rel="stylesheet" href="style.css">
 <script src="script.js"></script>
 
+<!-- Load React. -->
+<!-- Note: when deploying, replace "development.js" with "production.min.js". -->
+<script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
+<script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+<!-- Load React components -->
+<script type="module" src="./js/common/react/jsx-processed/Navbar.js"></script>
+
```

Comment:
Nice! One thing to note here (not such a big deal) is that it is common practice to load scripts like this at the end of the body, so after the header and button below. The reason being:

> one of the most important things to remember is that whenever the browser encounters a script tag, the DOM construction is paused! The entire DOM construction process is halted until the script finishes executing.
https://blog.logrocket.com/how-browser-rendering-works-behind-the-scenes-6782b0e8fb10/

It's definitely very minor and inconsequential for us, but it might make the home page feel that much more responsive if these script tags were defined at the bottom of the body.

Reply:
Thank you for sharing this convention and the logic behind it!